lost-sample/src/main/java/com/example/lost/MultipleLocationListenerSingleClientActivity.java

```diff
@@ -0,0 +1,156 @@
+package com.example.lost;
+
+import com.mapzen.android.lost.api.LocationListener;
+import com.mapzen.android.lost.api.LocationRequest;
+import com.mapzen.android.lost.api.LocationServices;
+import com.mapzen.android.lost.api.LostApiClient;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.os.Bundle;
+import android.support.v4.app.ActivityCompat;
+import android.widget.BaseAdapter;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Demonstrates one {@link LostApiClient}s receiving location updates at difference intervals
+ */
+public class MultipleLocationListenerSingleClientActivity extends ListActivity {
+
+  private static final int LOCATION_PERMISSION_REQUEST = 1;
+
+  LostApiClient lostApiClient;
+  LostApiClient otherLostApiClient;
```

Comment:
This is the single client example but there are still two clients here?

Given the multiple client activity/fragment example I think it would be more useful to test a single instance of the location client that makes multiple location requests here. What do you think?